testing/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AndroidPluginsSmokeTest.groovy

```diff
@@ -57,6 +58,8 @@ class AndroidPluginsSmokeTest extends AbstractPluginValidatingSmokeTest implemen
     }
 
     @UnsupportedWithConfigurationCache
+    @Ignore("AGP should use Provider API for Convention Task")
```

Comment:
🤔 I guess it's fine since 9.0 is a major

Reply:
What we need to do to not ignore this tests? AGP is one of the most important plugins.

Do we need to revert upgrade of `ConventionTask.render`?